mkdocs.yml

```diff
@@ -161,4 +161,8 @@ nav:
     - Resources: 
         - Learning: docs/resources/README.md
         - Workshops: docs/resources/workshops/README.md
+    - Releases: 
+        - 1.0.0-rc1: docs/releases/1.0.0-rc1.md
+        - 1.0.0-beta3: docs/releases/1.0.0-beta3.md
+        - 1.0.0-beta2: docs/releases/1.0.0-beta2.md
```

Comment:
Are these intentionally missing?

Reply:
There was no tags before 1.0.0-beta1 so the tool could not generate the changelog (It needs the previous and next release as arguments), so I tagged a commit as 1.0.0-alpha and managed to generate it:
https://docs.orchardcore.net/en/ag-releasenotes/docs/releases/1.0.0-beta1/